src/main/java/seedu/address/logic/commands/FindItemCommand.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.item.NameMatchesKeywordsPredicate;
+
+/**
+ * Finds and lists all items in item list whose name contains any of the keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindItemCommand extends Command {
+
+    public static final String COMMAND_WORD = "find";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all items whose names contain any of "
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
+            + "Example: " + COMMAND_WORD + " alice bob charlie";
```

Comment:
Will there be an item named alice bob charlie?

Reply:
changed to "apple banana carrot"